files/lpasspw

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo -n "$*: " >/dev/null
+answer=`cat $LPASS_HOME/pw`
```

Comment:
Great idea. Actually, it's probably best if the module has a 'noninteractive' flag only sets the LASTPASS_ASKPASS if that's true. Then the default prompting behavior will work for interactive users. I'll have to think about that.